omero/sysadmins/dropbox.txt

```diff
@@ -45,11 +45,11 @@ the following more specific requirements:
     Setting the property :property:`omero.fs.platformCheck` to ``False``
     in,
     
```

Comment:
Sorry I meant line-breaking  at 'or using...' **instead of before the file name** so it goes:
"Setting the property :property:`omero.fs.platformCheck` to ``False`` in :file:`etc/grid/templates.xml`
or using :omerocmd:`config set`